internal/os/smb/api.go

```diff
@@ -0,0 +1,47 @@
+package smb
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"k8s.io/klog"
+	//"k8s.io/utils/keymutex"
```

Comment:
Can we please delete it if its not used ?

Reply:
fixed